airflow/utils/helpers.py

```diff
@@ -244,3 +244,18 @@ def build_airflow_url_with_query(query: Dict[str, Any]) -> str:
     view = conf.get('webserver', 'dag_default_view').lower()
     url = url_for(f"Airflow.{view}")
     return f"{url}?{parse.urlencode(query)}"
+
+
+def exactly_one(*args):
+    """
```

Comment:
Challenge accepted:

How about

`return sum(map(bool,args) == 1`

Reply:
we might have a winner

Reply:
ok tests pass you win the prize.

nothing like a little code golf now and  again

Reply:
Though it's less efficient in certain cases :)

Reply:
yeah mine will fail faster but yours is cleaner 🤷

Reply:
But let me be honest - I cheated: https://stackoverflow.com/questions/22798938/how-to-check-if-exactly-one-value-in-an-iterable-resolves-to-true